data/remnant/remnant 2 cognizance.txt

```diff
@@ -1618,3 +1618,13 @@ mission "Remnant: Cognizance 38"
 			`	As you reach the end of your report, one of the Remnant nods in approval. "We are starting to disseminate your report along with the data Dusk and Gavriil collected. Both copies of the data match, and have passed the initial verifications, so it appears they were not affected by your departing... shift."`
 			`	They look around at the surrounding Remnant, and you have the sudden feeling that there is an entire level of discussion happening around you. You spot Aeolus moving through the crowd to the front, and he gestures with tones of appreciation. "Thank you for all your work on this, <first>. We have a strong appreciation for the risks you have taken alongside us, and I hope we will continue to explore together. That being said, I suspect it will take much collective effort to determine our next course of action." He looks around, and then winks at you. "In the meantime, I have unlocked access to a few more bits of rarer Korath salvage, including access to captured Chasers. It is not much, but it may give you a little more ease of outfitting the <ship>.`
 
+
+mission "Remnant: Cognizance 39"
```

Comment:
Good catch. The mission is now invisible. That being said, the self-fail isn't needed, so I'm not doing it in order to avoid cluttering things with unnecessary fails. They're a valuable tool, but everytime there's a fail in the mission list it gives people panic attacks. In any case, it's fine if the mission takes a few days to be completed.